lib/archethic/transaction_chain/mem_tables_loader.ex

```diff
@@ -54,6 +54,8 @@ defmodule Archethic.TransactionChain.MemTablesLoader do
     :ok = handle_pending_transaction(tx)
     :ok = handle_transaction_recipients(tx)
 
+    # TO DO: fix bug here
+    # tx_address == nil ||
```

Comment:
Do you have a example case where the transaction address is nil ? It normally should never be the case

Reply:
During node reboot

Reply:
I don't get this error, is it when the node restart juste after crashing or after some time ? It appears with multuple nodes or only one ?

Reply:
If you still have this error, you can remove the comment and create an issue for it please

Reply:
-Sure sir  I will remove it